packages/flutter/test/material/dropdown_test.dart

```diff
@@ -1296,7 +1296,7 @@ void main() {
     ));
 
     // By default the hint contributes the label.
-    expect(tester.getSemantics(find.byKey(key)), matchesSemantics(
+    expect(tester.getSemantics(find.text('test')), matchesSemantics(
```

Comment:
why this and below changes?

Reply:
because semantic menu button and semantic meun item become different semantics nodes after the change.